projects/lib-magnetar-quill/README.md

```
@@ -4,260 +4,171 @@
 ## **Table of Contents** 📚
-- [Features](#features)
-- [Installation](#installation)
-- [Quick Start](#quick-start)
-- [Usage](#usage)
-- [Plugin Architecture](#plugin-architecture)
-- [Theming & Customization](#theming--customization)
-- [Available Commands](#available-commands)
-- [Roadmap](#roadmap)
-- [Contributing](#contributing)
-- [License](#license)
+- [Status](#status-🏗️)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)